source/client/output_collector_impl.cc

```diff
@@ -122,5 +122,51 @@ std::string YamlOutputCollectorImpl::toString() const {
   return Envoy::MessageUtil::getYamlStringFromMessage(toProto(), true, true);
 }
 
+DottedStringOutputCollectorImpl::DottedStringOutputCollectorImpl(Envoy::TimeSource& time_source,
+                                                                 const Options& options)
+    : OutputCollectorImpl(time_source, options) {}
+
+std::string DottedStringOutputCollectorImpl::toString() const {
```

Comment:
As per my comment on the other review, can we make this a pure function that takes a Nighthawk proto output and performs the format? That way we can avoid putting this on the wire and allow clients to decide how they want to do the output formatting.

Reply:
So as we are planning to have [a separate CLI](https://github.com/envoyproxy/nighthawk/pull/159) instead, I think we can pursue landing this as-is, correct?